fixtures/generated/series.ts

```diff
@@ -111,7 +111,7 @@ export const series = {
 			linkText:
 				'The Flight Attendant review – strap in for a first-class murder mystery',
 			showByline: false,
-			byline: 'Lucy Mangan',
+			byline: 'Ellen E Jones',
```

Comment:
Was this a manual decision to strip Lucy Mangan of the credit here? Shocking

Reply:
This one is autogenerated. Clearly someone decided to change up the author after the fact

Reply:
🙀